packages/mockserver-config-writer/src/app-info.ts

```diff
@@ -1,5 +1,16 @@
 import type { Manifest, ManifestNamespace } from '@sap-ux/project-access';
 
+/**
+ * Get the main service data source name from manifest.json.
+ *
+ * @param manifest - Parsed manifest.json
+ * @returns - data source name from manifest.json
+ */
+function getMainServiceDataSourceName(manifest: Manifest): string | undefined {
+    const modelName = manifest['sap.ovp']?.globalFilterModel ?? '';
+    return manifest['sap.ui5']?.models?.[modelName] ? manifest['sap.ui5'].models[modelName].dataSource : undefined;
```

Comment:
Isn't  the result of`return manifest['sap.ui5']?.models?.[model] ? manifest['sap.ui5'].models[model].dataSource : undefined;` exactly the same as `return manifest['sap.ui5'].models[model]?.dataSource;`